fpga/src/main/scala/vcu118/Configs.scala

```diff
@@ -63,6 +69,37 @@ class WithVCU118Tweaks extends Config(
   new freechips.rocketchip.subsystem.WithNMemoryChannels(1)
 )
 
+class WithVCU118TweaksBareMetal extends Config(
```

Comment:
Can we drop the "BareMetal" naming, and name it `UARTTSIVCU118` in all places. The "baremetal" terminology I feel does not accurately describe what the additional capability of these configs is.